### ISSUE TITLE
[Question] How to wait for page loading in playwright python?

### ISSUE THREAD

**Issue by hhroh** [microsoft/playwright#14213]
How to wait for page loading in playwright python?
I want to use Selenium's implicitly_wait in playwright.

**Comment by rwoll** [comment]
See https://playwright.dev/python/docs/actionability. Playwright auto-waits by default. Does this help?

**Comment by rwoll** [comment]
Closing as part of triage! If the above doesn't answer your question, please file a new issue referencing this one!
